### ISSUE TITLE
Pickaxe still broken even with Unbreakable:1

### ISSUE THREAD

**Issue by SaitDev** [kicjow/Crazy-Enchantments#189]
I create a item with `Unbreakable:1` by _/give_ command. All enchant that pickaxe has are:
- Vanilla: Fortune, Efficiency
- CE: Furnace, Experience

The item still be lost after a while though the durability doesnt decrease.

**Comment by kicjow** [comment]
Never heard of Unbreakable:1. It must not show the damage. Its probably from furnace because I never heard of Unbreakable. So I dont have something to protect it.

**Comment by SaitDev** [comment]
Please try it: `/give @p diamond_pickaxe 1 0 {Unbreakable:1}` with furnace and experience. Ah, maybe wood pickaxe or stone for quicker broken testing. I guess furnace broke it, https://github.com/kicjow/Crazy-Enchantments/blob/master/src/me/BadBones69/CrazyEnchantments/Enchantments/PickAxes.java#L321

Sorry, i dont have dakataanticheat to try fixing it on my own, a bit hard for me to build

**Comment by kicjow** [comment]
Ill look into it when I get the chance.

**Comment by SaitDev** [comment]
Documents about Unbreakable can be found here https://hub.spigotmc.org/javadocs/spigot/org/bukkit/inventory/meta/ItemMeta.Spigot.html

Please, add this line right after line 814 https://github.com/kicjow/Crazy-Enchantments/blob/master/src/me/BadBones69/CrazyEnchantments/Methods.java#L814
`if (item.getItemMeta().isUnbreakable()) return;`

then Could you plz build a dev build for me? I really wanna this hot fix for my server. Sorry I dont have some premium plugin that crazy enchantmen supported, so its hard for me to build myself

Thank you.

**Comment by kicjow** [comment]
Don't worry. It will be in the next update later today.

**Comment by kicjow** [comment]
Just added it and it will be in the next update later today. Also so you know that Unbreak check method is 1.11.2 only. I made a checker so it should work fine but we shall see.

**Comment by SaitDev** [comment]
Glad to hear it. As far i know Unbreakable:1 was add in 1.8.x or 1.7.x, not remember exactly. But I just checked 1.8 java doc https://hub.spigotmc.org/nexus/content/groups/public/org/spigotmc/spigot-api/1.8-R0.1-SNAPSHOT/
Its `ItemMeta.spigot().isUnbreakable()` so I think this will work for older spigot version than 1.11.2
`if (item.getItemMeta().spigot().isUnbreakable()) return;`

**Comment by kicjow** [comment]
Yes, well I did know it was in the .spigot() but you got to remember thats for servers using spigot and not bukkit.

**Comment by SaitDev** [comment]
I see. Waitting for the next update 👍
